client/src/Components/Questions.js

```diff
@@ -0,0 +1,162 @@
+import React, { useState, useEffect } from "react";
+import { Button, Form, FormGroup, Label, Input } from "reactstrap";
+import dayjs from "dayjs";
+
+const Questions = (props) => {
+
+	const [i, setI] = useState(0);
```

Comment:
I would rename this something more descriptive, like `currentQuestionIndex`.

Reply:
Yes, I agree with you